gitlab/resource_gitlab_deploy_token.go

```diff
@@ -176,6 +184,19 @@ func resourceGitlabDeployTokenRead(d *schema.ResourceData, meta interface{}) err
 				if scope == "read_registry" {
 					d.Set("scopes.read_registry", true)
 				}
+
```

Comment:
I think at this point scope should be done as `switch` not a set of `if` conditions

Reply:
It could be since begining. But fixed.